src/components/index/Header.js

```diff
@@ -96,12 +96,11 @@ const Header = () => (
                         <div className="banner">
                             <img className="banner_image" src="https://lh5.googleusercontent.com/rYzgav3AjXGZctmse2MXMZb75T_moKq9rScNd_oRFKtDX8dB0MUSJ0KPu2E6rrISs4jwOtX8ESkFCnXwZYTY2BScU0cOvwrlQsZdmbwH7fJA2M0VywmydZPTFFfsLm9XAQ=w1600" alt="image_1" style={{ width: '70%' }}></img>
                             <h1 style={{ fontSize: '2.2rem' }}>
-                            The Call for Presentations is now OPEN for the 2021 TheiaCon Event!
-                            Submit your proposal today to be a speaker at the largest Theia event of the year!
+                            TheiaCon is a virtual conference focused around the Eclipse Theia IDE ecosystem. It brings together a diverse group of Theia developers, adopters, and other contributors. The program will feature a mix of full-length talks, expert panel discussions and short lightning talks focused on project insider, adopter stories, and broader ecosystem.
```

Comment:
Overall the changes look good, I'm just not sure what "project insider" is. Maybe the sentence needs to be adjusted:

```
...talks focused on project insider...
```